deployment/context/versions.template.mustache

```diff
@@ -2,15 +2,15 @@ import { Versions as VersionsInterface } from "../../../deployment/context/inter
 
 export class VersionsClass implements VersionsInterface {
   getSchemaVersion(): string {
-    return "{{ versions.schema }}";;
+    return "{{ versions.schema }}";
```

Comment:
Was this broken?

Reply:
Not sure how that came about. I'll take a look.